hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconConstants.java

```diff
@@ -58,4 +58,9 @@ private ReconConstants() {
       + PIPELINE_DB_SUFFIX;
   public static final String RECON_SCM_NODE_DB =
       "recon-node.db";
+  public static final long MAX_FILE_SIZE_UPPER_BOUND = 1125899906842624L;
+  public static final long MIN_FILE_SIZE_UPPER_BOUND = 1024L;
+  public static final int NUM_OF_BINS = (int)(Math.log(
+          MAX_FILE_SIZE_UPPER_BOUND) / Math.log(2) - Math.log(
+                  MIN_FILE_SIZE_UPPER_BOUND) / Math.log(2)) + 1;
```

Comment:
```suggestion
  // 41 bins
  public static final int NUM_OF_BINS = (int) Math.ceil(Math.log(
      (double) MAX_FILE_SIZE_UPPER_BOUND / MIN_FILE_SIZE_UPPER_BOUND) /
      Math.log(2)) + 1;
```
Add a Math.ceil() just in case